backend-beeyond/src/main/java/at/htl/beeyond/resource/ApplicationResource.kt

```diff
@@ -75,9 +75,7 @@ class ApplicationResource {
                 ?: return Response.status(Response.Status.NOT_FOUND).build()
 
         if(application.status == ApplicationStatus.PENDING){
```

Comment:
Man kann auch approven, wenn die Application in Status "DENIED" ist.
Beim deny auch.